runtime/tests/redirector/jep178/module.xml

```diff
@@ -34,7 +34,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			<makefilestub data="CFLAGS += -Wc,DLL,EXPORTALL">
 				<include-if condition="spec.zos_390.*"/>
 			</makefilestub>
-			<makefilestub data="UMA_CC_MODE += -qpic -brtl -bexpall">
+			<makefilestub data="UMA_CC_MODE += -fpic -brtl -bexpall">
```

Comment:
This can't be changed unconditionally.
FYI module.xml (UMA) is mostly used for IBM Java 8 builds, but it's still (was) possible use UMA to build OpenJ9 versions, It's something we should check, if UMA is still working for Java 23 with OpenXL. By default we use cmake, but UMA can be used if adding options for debugging purposes.